src/app/board/board.component.ts

```diff
@@ -6,6 +6,8 @@ import { Observable, Subscription } from 'rxjs';
 import { BoardsService } from '@app/core/services';
 import { List } from '@app/core/models';
 import { listTitleValidationErrors } from '@app/core/constants';
+import { MatDialog } from '@angular/material/dialog';
+import { DialogModalComponent } from '@app/shared/dialog-modal/dialog-modal.component';
```

Comment:
Please add material import to the core ones
Add barrel import for DialogModalComponent